settings/FixedDueDateScheduleForm.js

```diff
@@ -143,28 +154,32 @@ Delete
   }
 
   renderPaneTitle() {
-    const { initialValues, stripes: { intl } } = this.props;
-    const selectedSet = initialValues || {};
+    const {
+      initialValues: selectedSet = {},
```

Comment:
I think this will override the `initialValues` which changes the previous implementation. Could you please change it back to what it was? or just do:

`const selectedSet = this.props.initialValues || {}`

We have to be careful not to mutate `initialValues`. redux-form is sensitive about that object.

Reply:
Hi Michał, you probably haven't mentioned that in this particular case I have applied ES6 object destructuring pattern. You can have a look of the docs [here](https://javascript.info/destructuring-assignment). In this case I believe we don't need to create a new variable as we can distructure it directly from this.props object with the default value.

Reply:
Oh you are right @anastasiaHahanova! Sorry about that. Somehow it was hard spot that for me.